src/__private_api.rs

```diff
@@ -18,7 +18,7 @@ mod sealed {
 
 // Types for the `kv` argument.
 
-impl<'a> KVs<'a> for &'a [(&'a str, Value<'a>)] {
+impl<'a, const N: usize> KVs<'a> for &'a [(&'a str, Value<'a>); N] {
```

Comment:
I'm fine with the change. I do wonder if this changes the binary size as now create code per `N`/size, but I don't know if this will impact anything.